other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,50 @@
+# Getting help with Altis.
+
+If you need help, this guide will show you how to most effectively get you the help you need.
+
+While we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
+
+<img width="1398" alt="Altis Dashboard support link" src="https://user-images.githubusercontent.com/1263370/113014595-a119f980-9174-11eb-95d6-0e2a9718c5e5.png">
+
+We break down support into different types. Each has their own language and processes, but they're somewhat intuitive once you internalize the language a little. Let's go over the types.
+
+<img width="1282" alt="Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+
+We currently have four types of support. Below is a description of each and how they are different from each other.
+
+# Incident
+
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an incident, and requires investigation.
+
+From the support side, Incidents are the things that make us scamble to respond and jump on the problem. If the site is down, this is an Incident, with a prioriy of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means neccessary.
```

Comment:
Not sure about the wording here around "scrambling to respond", the image in my head is of a panicky disorganised response which we don't do. Maybe something like:
```suggestion
From the support side, Incidents get a rapid response. If the site is down, this is an Incident, with a priority of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
```